concent_api/core/tests/utils.py

```diff
@@ -979,6 +979,12 @@ def _pass_rpc_synchronization(self, _rpc, _address, _tx_sign):  # pylint: disabl
     def is_account_status_positive_true_mock(self, client_eth_address, pending_value):  # pylint: disable=unused-argument, no-self-use
         return True
 
+    def claim_deposit_true_mock(self, concent_use_case, requestor_ethereum_address, provider_ethereum_address, subtask_cost):  # pylint: disable=unused-argument, no-self-use
+        return True, True
```

Comment:
Please add brackets to this return statement

Reply:
:+1: done